### PR TITLE
Minor supercruise fixes and objective payout changes.

### LINKED ISSUE #7341: Vortex Rifles require piercing
Vortex rifles have incorrect piercing flags on the bullet and cannot penetrate mobs.
They need to act the same as penetrator rifles.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Some minor changes to supercruise that wasn't worth including in the major update.

 - Objective payouts have been changed. Their ranges now all intersect, making it not just the same assassination mission that will have the best payout.
 - Vortex rifle has been fixed.
 - Blob ruin generator has been fixed.

## Why It's Good For The Game

Fixes some bugs, better payouts.

## Testing Photographs and Procedure

image

image

image

## Changelog
:cl:
fix: Vortex rifles can now damage mobs
fix: Blob ruin generator has been fixed
tweak: Objective payout ranges have been adjusted to always intersect with each other.
/:cl:

### CODE REVIEW

File: code/modules/shuttle/super_cruise/orbital_poi_generator/loot/vortex_rifle.dm [hunk 1]
```diff
@@ -4,3 +4,8 @@
 	icon_state = "vortex"
 	item_state = null
 	ammo_type = list(/obj/item/ammo_casing/energy/vortex)
+
+/obj/item/gun/energy/vortex/examine(mob/user)
+	. = ..()
+	if(isabductor(user))
+		. += "<span class='abductor'>This shouldn't be here... It's not from this era...</span>"
```
Comment (Archanial) [inline code/modules/shuttle/super_cruise/orbital_poi_generator/loot/vortex_rifle.dm]: Why?

**Review by qwertyquerty — DISMISSED** [review]
Looks good code wise

File: code/modules/shuttle/super_cruise/orbital_poi_generator/loot/vortex_rifle.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/modules/shuttle/super_cruise/orbital_poi_generator/loot/vortex_rifle.dm]: Lore

Comment (Archanial) [reply code/modules/shuttle/super_cruise/orbital_poi_generator/loot/vortex_rifle.dm]: What lore? It sounds stupid

Comment (PowerfulBacon) [inline code/modules/shuttle/super_cruise/orbital_poi_generator/loot/vortex_rifle.dm]:
```suggestion
		. += "<span class='abductor'>It has a subspace power core installed.</span>"
```

**Review by Archanial — APPROVED** [review]

**Pull request merged**